.github/workflows/build-test-distribute.yaml

```diff
@@ -71,34 +71,55 @@ jobs:
           make dev/tools
       - run: |
           make test
-  distributions:
-    timeout-minutes: 40
-    needs: ["check", "test", "test_e2e", "test_e2e_env"]
-    if: ${{ always() }}
+  metadata:
```

Comment:
NIT: if you have outputs in `build` as well I'd collapse these 2 actions (it makes the graph a little simpler to follow)

Reply:
done. removed `metadata` job